gateway/src/apicast/policy/soap/soap.lua

```diff
@@ -90,6 +91,13 @@ end
 function _M:rewrite(context)
   local soap_action_uri = extract_soap_uri()
 
+  -- If the soap_action_uri is a complete url, the mapping rule is only the
```

Comment:
I'm missing something. How is this change in the SOAP policy related to the original issue?

Reply:
The action in the content-type can be a full url, so we need to pass it correctly: 

https://github.com/3scale/APIcast/blob/1266bc8dda609093958ef2b1c8592cd539f5a129/spec/policy/soap/soap_spec.lua#L112-L115